web/viewer.html

```diff
@@ -97,24 +97,35 @@
       </div>  <!-- sidebarContainer -->
 
       <div id="mainContainer">
-        <div class="findbar hidden doorHanger hiddenSmallView" id="findbar">
-          <label for="findInput" class="toolbarLabel" data-l10n-id="find_label">Find:</label>
-          <input id="findInput" class="toolbarField" tabindex="91">
-          <div class="splitToolbarButton">
-            <button class="toolbarButton findPrevious" title="" id="findPrevious" tabindex="92" data-l10n-id="find_previous">
-              <span data-l10n-id="find_previous_label">Previous</span>
-            </button>
-            <div class="splitToolbarButtonSeparator"></div>
-            <button class="toolbarButton findNext" title="" id="findNext" tabindex="93" data-l10n-id="find_next">
-              <span data-l10n-id="find_next_label">Next</span>
-            </button>
+        <div class="findbar hidden doorHanger" id="findbar">
+          <div id="findbarLabelContainer">
+            <label for="findInput" class="toolbarLabel" data-l10n-id="find_label">Find:</label>
```

Comment:
Looking through my local patches/diffs, I actually found this https://gist.github.com/Snuffleupagus/cbba3a6823c3a02a59a1a1d8bcbe6059. As far as I can find I've never submitted a PR for that, but if memory serves it should work just fine.